test/capz/npm/npm-e2e.sh

```diff
@@ -255,7 +255,7 @@ run_npm_conformance () {
         mkdir npm-kubernetes
         cd npm-kubernetes
         # NOTE: if this is not downloaded every run, then probably need to sleep before the VFP tag verification
-        git clone https://github.com/huntergregory/kubernetes.git --depth=1 --branch=quit-on-failure
+        git clone https://github.com/huntergregory/kubernetes.git --depth=1 --branch=quit-on-failure # Change this as well?
```

Comment:
we can leave this one as is. This file is more so a reference on how to verify Windows builds on CAPZ clusters